README.md

```diff
@@ -121,8 +121,18 @@ First of all, thanks for wanting to contribute to DevStream! For more details on
 
 We will regularly organize `DevStream Community Meeting`, please visit the [wiki](https://github.com/devstream-io/devstream/wiki) page for details.
 
-- Message us on <a href="https://cloud-native.slack.com/archives/C03LA2B8K0A" target="_blank">Slack.</a>
-- For Chinese users, the WeChat group QR code is as below:
+Please join our Slack channel. Here's how:
+
+1. [Invite yourself to CNCF's Slack if you haven't done so](https://slack.cncf.io). 
+    - Input your email address, and click "get my invite."
```

Comment:
Should we change the "get my invite." to "get my invite". ?

Same as below.

Reply:
Amerian English recommends putting the trailing punctuation within the closing quote. I recommend everyone who wants to contribute to English documentation have the "Grammarly" browser plugin as well as [this book](https://www.amazon.com/Dreyers-English-Utterly-Correct-Clarity/dp/0812995708).